comparison/loadtest_test.go

```diff
@@ -9,40 +9,61 @@ import (
 )
 
 func TestBuildLoadDBDumpCmd(t *testing.T) {
-	newIP := "192.168.1.1"
-	oldIPs := []string{"54.78.456.5", "56.78.98.1"}
-
-	cmds, err := buildLoadDBDumpCmds("dbfilename", newIP, oldIPs, dbSettings{
-		UserName: "mmuser",
-		Password: "mostest",
-		DBName:   "mattermost",
-		Host:     "test-db-0.c0gzspkyf2r5.us-east-1.rds.amazonaws.com",
-		Engine:   "aurora-postgresql",
+	t.Run("without permalink IPs to replace", func(t *testing.T) {
+		newIP := "192.168.1.1"
+		oldIPs := []string{}
+
+		cmds, err := buildLoadDBDumpCmds("dbfilename", newIP, oldIPs, dbSettings{
+			UserName: "mmuser",
+			Password: "mostest",
+			DBName:   "mattermost",
+			Host:     "test-db-0.c0gzspkyf2r5.us-east-1.rds.amazonaws.com",
+			Engine:   "aurora-postgresql",
+		})
+		require.NoError(t, err)
+
+		// Check the slice contains only two commands: zcat and psql
+		require.Len(t, cmds, 2)
+		require.True(t, strings.HasPrefix(cmds[0], "zcat"))
+		require.True(t, strings.HasPrefix(cmds[1], "psql"))
+	})
```

Comment:
This is the only new test in this file. The other part of the diff contain only whitespace changes.